app/javascript/App/components/ItemSelectorAlt.js

```diff
@@ -2,19 +2,25 @@ import React, { Component } from 'react'
 import Type from 'prop-types'
 import Icon from './Icon'
 
+const inputStyle = {
+  width: 'auto',
+  marginRight: '.5rem'
+}
+
 export default class ItemSelectorAlt extends Component {
   selectedItems = () => this.props.selectedItems || []
   isSelected = (item) => {
-    const selected = this.selectedItems().map((item) => JSON.stringify(item))
-    return selected.includes(JSON.stringify(item))
+    const selected = this.selectedItems().feedSources && this.selectedItems().feedSources.length && this.selectedItems().feedSources.map((item) => JSON.stringify(item))
```

Comment:
feedSources is too specific, this component should be reusable

Reply:
i'll reply to this one in comments of the PR